### PR TITLE
Cleaning up azure devops in spk-config

### PR DESCRIPTION
**Pull request by mtarng**

### CODE REVIEW

**Review by evanlouie — APPROVED** [review]

File: spk-config.yaml [hunk 1]
```diff
@@ -6,12 +6,11 @@ azure_cli:
 
 # Azure DevOps Configuration
 azure_devops: # Should this live under the `spk` configuration?
-  access_token: "access-token" # This is your Personal Access Token with permission to modify and access this private repo. Ex. "${env:AZDO_PERSONAL_ACCESS_TOKEN}". Leave this empty if project is public
-  hld_repository: "my-hld-repository"
-  manifest_repository: "my-manifest-repository"
-  org: "my-az-do-org"
-  project: "my-mono-repo-project"
-  server_url:
+  access_token: "hpe3a9oiswgcodtfdpzfiek3saxbrh5if1fp673xihgc5ap467a" # This is your Personal Access Token with permission to modify and access this private repo. Leave this empty if project is public
```
Comment (jmspring) [inline spk-config.yaml:9]: how do I set up a personal access token?

**Review by jmspring — APPROVED** [review]
lgtm

File: spk-config.yaml (hunk 1 above)
Comment (mtarng) [reply spk-config.yaml:9]: `https://dev.azure.com/<yourusername>/_usersSettings/tokens`

You can also get there by clicking on your profile and selecting `Azure DevOps profile` -> Security -> Personal access tokens.

Or do you mean to include these instructions as a part of this?

File: src/commands/mocks/spk-config.yaml [hunk 2]
```diff
@@ -6,11 +6,11 @@ azure_cli:
...
 azure_devops: # Should this live under the `spk` configuration?
-  org: "my-az-do-org"
-  project: "my-mono-repo-project"
-  hld_repository: "my-hld-repository"
-  manifest_repository: "my-manifest-repository"
-  access_token: "access-token" # This is your Personal Access Token with permission to modify and access this private repo. Ex. "${env:AZDO_PERSONAL_ACCESS_TOKEN}". Leave this empty if project is public
+  access_token: "hpe3a9oiswgcodtfdpzfiek3saxbrh5if1fp673xihgc5ap467a" # This is your Personal Access Token with permission to modify and access this private repo. Leave this empty if project is public
+  hld_repository: "dev.azure.com/bhnook/fabrikam/_git/hld" # Repository URL for your Bedrock HLDs
+  manifest_repository: "dev.azure.com/bhnook/fabrikam/_git/materialized" # Repository URL that is configured for flux. This holds the kubernetes manifests that is generated by fabrikate.
+  org: "https://dev.azure.com/bhnook" # Your AzDo Org
```
Comment (andrebriggs) [inline src/commands/mocks/spk-config.yaml:12]: @evanlouie @mtarng we should just use `bhnook` here

Comment (samiyaakhtar) [reply src/commands/mocks/spk-config.yaml:12]: yes, specifying a URL in this one is going to break introspection

File: spk-config.yaml [hunk 3]
```diff
@@ -6,12 +6,11 @@ azure_cli:
...
+  hld_repository: "dev.azure.com/bhnook/fabrikam/_git/hld" # Repository URL for your Bedrock HLDs
+  manifest_repository: "dev.azure.com/bhnook/fabrikam/_git/materialized" # Repository URL that is configured for flux. This holds the kubernetes manifests that is generated by fabrikate.
+  org: "https://dev.azure.com/bhnook" # Your AzDo Org
```
Comment (samiyaakhtar) [inline spk-config.yaml:12]: Same here

**Pull request merged**